test/integration/targets/inventory_aws_conformance/runme.sh

```diff
@@ -0,0 +1,168 @@
+  ec2_dns_name: public_dns_name
+  ec2_group_name: placement.group_name
+  ec2_id: instance_id
+  ec2_instance_profile: iam_instance_profile | default("")
+  ec2_ip_address: public_ip_address
+  ec2_kernel: kernel_id | default("")
+  ec2_monitored:  monitoring.state in ['enabled', 'pending']
+  ec2_monitoring_state: monitoring.state
+  ec2_account_id: owner_id
+  ec2_placement: placement.availability_zone
+  ec2_ramdisk: ramdisk_id | default("")
+  ec2_reason: state_transition_reason
+  ec2_security_group_ids: security_groups | map(attribute='group_id') | list |  join(',')
+  ec2_security_group_names: security_groups | map(attribute='group_name') | list |  join(',')
+  ec2_state: state.name
+  ec2_state_code: state.code
+  ec2_state_reason: state_reason.message if state_reason is defined else ""
+  ec2_sourceDestCheck: source_dest_check | lower | string  # butchered snake_case case not a typo.
+
+  # vars that just need ec2_ prefix
+  ec2_ami_launch_index: ami_launch_index | string
+  ec2_architecture: architecture
+  ec2_client_token: client_token
+  ec2_ebs_optimized: ebs_optimized
+  ec2_hypervisor: hypervisor
+  ec2_image_id: image_id
+  ec2_instance_type: instance_type
+  ec2_key_name: key_name
+  ec2_launch_time: 'launch_time | regex_replace(" ", "T") | regex_replace("(\+)(\d\d):(\d)(\d)$", ".\g<2>\g<3>Z")'
+  ec2_platform: platform | default("")
+  ec2_private_dns_name: private_dns_name
+  ec2_private_ip_address: private_ip_address
+  ec2_public_dns_name: public_dns_name
+  ec2_region: placement.region
+  ec2_root_device_name: root_device_name
+  ec2_root_device_type: root_device_type
+  ec2_spot_instance_request_id: spot_instance_request_id | default("")
+  ec2_subnet_id: subnet_id
+  ec2_virtualization_type: virtualization_type
+  ec2_vpc_id: vpc_id
+  tags: dict(tags.keys() | map('regex_replace', '[^A-Za-z0-9\_]', '_') | list | zip(tags.values() | list))
+
+keyed_groups:
+  - key: '"ec2"'
+    separator: ""
+  - key: 'instance_id'
+    separator: ""
+  - key: tags
+    prefix: tag
+  - key: key_name | regex_replace('-', '_')
+    prefix: key
+  - key: placement.region
+    separator: ""
+  - key: placement.availability_zone
+    separator: ""
+  - key: platform | default('undefined')
+    prefix: platform
+  - key: vpc_id | regex_replace('-', '_')
+    prefix: vpc_id
+  - key: instance_type
+    prefix: type
+  - key: "image_id | regex_replace('-', '_')"
+    separator: ""
+  - key: 'security_groups | json_query("[].group_name") | map("regex_replace", "-", "_") | list'
+    prefix: security_group
+EOF
+
+rm -f "$OUTPUT_DIR/plugin.out"
+ANSIBLE_JINJA2_NATIVE=1 ansible-inventory -vvvv -i "$OUTPUT_DIR/test.aws_ec2.yml" --list --output="$OUTPUT_DIR/plugin.out"
+rm -f "$OUTPUT_DIR/aws_ec2.yml"
+rm "$TARGET/lib/boto3"
+rm "$TARGET/lib/botocore"
+rm -r "${OUTPUT_DIR}/aws-ec2-inventory/"
```

Comment:
None of this cleanup should be needed, since both the output and current working directory are automatically cleaned up at the end of the test.

Reply:
Correction, the temporary directory is deleted and created at the start of each test target.